Content.Client/CartridgeLoader/Cartridges/LogProbeUiFragment.xaml.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Content.Shared.CartridgeLoader.Cartridges;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.CartridgeLoader.Cartridges;
+
+[GenerateTypedNameReferences]
+public sealed partial class LogProbeUiFragment : BoxContainer
```

Comment:
Isnt this a normal control rather than a UIFragment (considering this doesnt inherit that)